install_all.sh

```diff
@@ -1,66 +1,69 @@
 #!/bin/bash
 
 SUDO=""
+VundleDir="$HOME/.vim/bundle/Vundle.vim"
```

Comment:
Referring to Google's [Shell Style Guide](https://google.github.io/styleguide/shell.xml#Naming_Conventions), the constant variable name should be capitalized but not camelCase, and use lower-case with underscores for normal variables. 

I personally like this naming convention more, so after you modified it I'll accept the pull request.
Anyway, thanks :)